src/main/java/io/antmedia/muxer/EndpointMuxer.java

```diff
@@ -242,7 +257,7 @@ public synchronized void clearResource() {
 	public synchronized boolean addVideoStream(int width, int height, AVRational timebase, int codecId, int streamIndex, boolean isAVC, AVCodecParameters codecpar) {
 		
 		boolean result = super.addVideoStream(width, height, timebase, codecId, streamIndex, isAVC, codecpar);
-		if (result) 
+		if (result && !this.format.equals("mpegts"))
```

Comment:
I think positive statement is more meaningful.

I mean run this line if it is `flv` because we are doing something specific related to the `flv` or `rtmp`